### PR TITLE
Add browser support section to contributing doc

### PR DESCRIPTION
**Pull request by anselmbradford**
## Additions

- Adds a generic browser support section to the contributing doc template.

### CODE REVIEW

**Comment by anselmbradford** [comment]
@jimmynotjim @ascott1

**Comment by Scotchester** [comment]
Seems a bit too generic. I think we should include either more details or a link to the browser support guide in cfpb/development.

**Comment by anselmbradford** [comment]
The templates in this repo are largely meant to be re-written (except for the headings), but have catch-all fallback language. With that in mind, what do you think it should say?

**Comment by jimmynotjim** [comment]
>  are largely meant to be re-written

🤣 I can't count how many readmes I've looked at that still have the default. I'm sure most of them eventually change, but that definitely isn't the case for many project's first few ~~commits~~ months

Should it more closely [match cfgov-refresh](https://github.com/cfpb/cfgov-refresh/blob/master/CONTRIBUTING.md#changing-the-codebase) since most projects will eventually be related in some way?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -25,8 +25,15 @@ guidance below.
...
+unit tests that validate implemented features and the presence or lack of defects.
 Additionally, the code should follow any stylistic and architectural guidelines
 prescribed by the project. In the absence of such guidelines, mimic the styles
 and patterns in the existing code-base.
+
+
+## Browser support
+
+We configure our build chain tools
+(typically [Autoprefixer](https://github.com/postcss/autoprefixer) and/or [Babel](https://babeljs.io)
```
Comment (Scotchester) [inline CONTRIBUTING.md]:
```suggestion
(typically [Autoprefixer](https://github.com/postcss/autoprefixer)
and [Babel](https://babeljs.io))
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -25,8 +25,15 @@ guidance below.
...
+to support a reasonable set of backward compatibility with older browsers.
```
Comment (Scotchester) [inline CONTRIBUTING.md]:
```suggestion
to support a reasonable set of backward compatibility with older browsers.

Please read up on
[our current browser support guidance](https://github.com/cfpb/development/blob/master/guides/browser-support.md)
and follow it when contributing to this project.
```

**Review by Scotchester — COMMENTED** [review]
OK, added some suggested language. What do you think?

File: CONTRIBUTING.md (hunk 1 above)
Comment (Scotchester) [reply CONTRIBUTING.md]: This one is just a formatting suggestion.

**Review by Scotchester — APPROVED** [review]

**Pull request merged**